plugins/modules/nsxt_rest.py

```diff
@@ -218,19 +218,27 @@ def execute(self):
 
         if self.method == "post" or self.method == "put" or self.method == "patch":
             before_resp = self.operate_nsxt(method="get", ignore_errors=True)
-            if before_resp:
+            before_revision = ""
+
+            if before_resp and "_revision" in before_resp :
```

Comment:
suggestion 

if before_resp:
    before_revision = before_resp.get('_revision', '')

This might simplify a little the logic